app/controllers/users_controller.rb

```diff
@@ -5,14 +5,13 @@ def new
 
   def show
     @user = User.find_by(id: params[:id])
-    if @user.blank?
-      redirect_to signup_path
-    end
+    redirect_to signup_path if @user.blank?
```

Comment:
chị chỉnh line code 8 một tí nhé
`return if @user`
`flash[:warning] = "User not found"`
`redirect_to signup_path`